### DISCUSSION TITLE
Plotting not working

### DISCUSSION THREAD

**Discussion by hectormara** [pybamm-team/liionpack#339] [General]
There is a plotting issue that might be an issue for some users. When I run lp.plot_output(output) I get the graphs correctly. The problem comes at the second run. I change some parameters, the simulation runs fine but after that the plots do not show anymore. The only solution I found is to restart the kernel and import the libraries from scratch. Is it an issue with liionpack or with jupyter notebook, visual studio code?

**Comment by kratman** [comment]
This sounds more like a bug report than a discussions. So I moved it there
